doc_admin/AdminManual.adoc

```diff
@@ -21,11 +21,20 @@ pass:[<br>][.small]#Policies is made available under a http://creativecommons.or
 
 == 1 Abstract
 
-This application is the first of a set of software products that ISFfootnote:[Informatici Senza Frontiere = Computer scientists without frontiers] has developed to support the management and the activities of the St. Luke Hospital in Angal (Uganda). After that initial mission, the St. Luke Hospital has become the first one of a long list of hospitals that have found this software useful.
+[Open Hospital][openhospital] (OH) is a free and open-source Electronic Health Record (EHR) software application.
+Open Hospital is deployed as a desktop application that can be used in a standalone, single user mode (PORTABLE mode)
+or in a client-server network configuration (CLIENT mode), where multiple clients and users can connect to the same database server.
```

Comment:
`users can connect to the same database server` -> `users connect to the same database server`  (remove `can`)

Reply:
Done !